fmod-oxide/src/core/system/callback.rs

```diff
@@ -113,7 +113,7 @@ pub enum Instance {
 }
 
 impl Instance {
-    fn from_raw(kind: c_uint, pointer: *mut c_void) -> Self {
+    fn from_raw(kind: FMOD_ERRORCALLBACK_INSTANCETYPE, pointer: *mut c_void) -> Self {
```

Comment:
Not sure if you had this take c_uint for a reason, but directly taking FMOD_ERRORCALLBACK_INSTANCETYPE seems like the easiest way to deal with the msvc difference here.

Reply:
I think I forgot when writing this code. Oops!